tests/e2e/tests/dashboard/dashboard_test.go

```diff
@@ -307,5 +362,5 @@ func getPodList(namespace string, selector string) ([]string, error) {
 }
 
 func allowPrometheusSync() {
-	time.Sleep(30 * time.Second)
+	time.Sleep(1 * time.Minute)
```

Comment:
nit: in the interest of sleeping as little as possible, can we have a polling loop here instead with a much smaller sleep?

Reply:
Possibly. The issue is trying to figure out when prometheus successfully scrapes the cAdvisor data for the Mixer container -- the existence of that metric is what is being checked for in the dashboard test itself...  Let's punt on this for the moment.